python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -72,13 +72,37 @@ def __init__(
 
         self.calibration_config: dict[str, dict[str, typing.Any]] = dict()
 
-    async def setup_electrometers(self) -> None:
-        """Setup all electrometers."""
+    async def setup_electrometers(
+        self, mode: int, range: float, integration_time: float
```

Comment:
if mode is an enumeration, please, use the enumeration instead of an int.

Reply:
I think we do pass an int that is then turned into an enum by the controller in ts_electrometer (at least, that's my understanding - @parfa30 can confirm).  Should I make a note of that? Should I call this an enum value?

Reply:
yeah, I will add in the enumeration!